cimr/processor/utils.py

```diff
@@ -90,3 +92,44 @@ def add_line_in_log():
     """Add an arbitrary divider to the log output for readability."""
     logging.info(' ' + '-' * 60 + '\n')
 
+
+def remove_palindromic(df):
+    """Removing palindromic variants from processed data."""
+    df = df.copy()
+    logging.debug(f'{df.head(2)}')
+    logging.debug(f'{df.columns}')
+    if 'ref' in df.columns:
+        df['g'] = df['alt'].str.upper() + df['ref'].str.upper()
+    elif 'effect_allele' in df.columns:
+        df['g'] = df['effect_allele'].str.upper() + df['non_effect_allele'].str.upper()
+    else:
+        logging.error(f' alleles need to be indicated.')
+        sys.exit(1)
+
+    df = df.loc[~((df.g == 'AT') | (df.g == 'TA') | (df.g == 'CG') | (df.g == 'GC'))]
+    return df.drop('g', axis=1)
+
+
+def try_convert(value, typing):
+    """Try converting input value to specific variable types,
+    e.g. int, float..
+    """
+    converted = True
+    try:
+        value = typing(value)
+    except:
+        converted = False
+    return converted
+
+
+def inferred(value):
```

Comment:
This function doesn't seem to be used anywhere yet. Do you plan to use it later?

By the way, if you only need to convert an input value to either `int` or `float`, you can combine `try_convert` and this function together into something like this, which might be more efficient:
```
def inferred(value):
    try:
        return int(value)
    except:
        try:
            return float(value)
        except:
            return value 
```

Reply:
Good points. I will add this in next PR. Thank you!